Library/Homebrew/utils/pypi.rb

```diff
@@ -41,6 +47,11 @@ def get_pypi_info(package, version)
   def update_python_resources!(formula, version = nil, print_only: false, silent: false,
                                ignore_non_pypi_packages: false)
 
+    if !print_only && AUTOMATIC_RESOURCE_UPDATE_BLOCKLIST.include?(formula.name)
```

Comment:
```suggestion
    if !print_only && AUTOMATIC_RESOURCE_UPDATE_BLOCKLIST.include?(formula.full_name)
```

In `brew irb`:

```
irb(main):001:0> Formulary.factory("brewsci/bio/dsh-bio").name
=> "dsh-bio"
irb(main):002:0> Formulary.factory("brewsci/bio/dsh-bio").full_name
=> "brewsci/bio/dsh-bio"
```

Note: `audit` seems to use `Dependency#name` which is different from `Formula#name`:
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb#L403

```
irb(main):010:0> Formulary.factory("brewsci/bio/abricate").deps[2].name
=> "brewsci/bio/any2fasta"
```

Reply:
I think this makes sense. I don't know if formulae that aren't in homebrew-core will be added to this list but it probably doesn't hurt to add the functionality in case that are in the future.